regulation-worker/internal/delete/api/api.go

```diff
@@ -76,6 +108,22 @@ func (api *APIManager) Delete(ctx context.Context, job model.Job, destConfig map
 		return model.JobStatusFailed
 	}
 
+	// Refresh Flow Start
+	if delAccountIdExists {
```

Comment:
Won't it be better to know, what would be the status code returned by the transformer in case of token expiry ( I guess it will be 401). Once, we know it, we can avoid checking for each user in the batch if token for it failed or not. And, directly decide based on the status code if we need to refresh to token or not?

Reply:
As discussed, we are moving ahead with current implementation of checking with an enum kind of thing that we get through the response from transformer rather than relying on source-codes